images/k8ssandra-system-logger/config/main.tf

```diff
@@ -0,0 +1,19 @@
+terraform {
+  required_providers {
+    apko = { source = "chainguard-dev/apko" }
+  }
+}
+
+module "accts" { source = "../../../tflib/accts" }
+
+output "config" {
+  value = jsonencode({
+    contents = {
+      packages = ["cass-operator-config", "vector", "libstdc++"]
```

Comment:
Given this particular image, i'm not sure if we'd ever need to switch out cass-operator-config, but perhaps to avoid needing to change in future, we can change this to the extra_packages approach?

i.e:

```
variable "extra_packages" {
  description = "The additional packages to install"
  default     = ["cass-operator-config"]
}
```

And then similarly in the config module:
```
packages  = concat(["libstdc++", "vector"], var.extra_packages)
```

Reply:
Or is vector the main package here?

Reply:
Correct, `vector` is the main package here.

> i'm not sure if we'd ever need to switch out cass-operator-config

The variables is compatible with the FIPS version of this image.